File: Common/Entity/src/main/java/it/ey/dto/AttivitaSensibileDTO.java
package it.ey.dto;


import it.ey.entity.EventoRischio;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class AttivitaSensibileDTO  extends  CampiTecniciDTO{

    private Long id;

    private Long idSezione23;

    private String denominazione;

    private String descrizione;

    private String processoCollegato;

    private List<EventoRischioDTO> eventoRischio;

       //CAMPI MONGO

    private AttoreDTO attore;

    private   UlterioriInfoDTO ulterioriInfo;
}

File: PIAO_FE/src/app/shared/models/classes/attivita-sensibile-dto.ts
import { AttoreDTO } from './attore-dto';
import { EventoRischiosoDTO } from './evento-rischioso-dto';
import { UlterioriInfoDTO } from './ulteriori-info-dto';

export class AttivitaSensibileDTO {
  id!: number;
  idSezione23!: number;
  denominazione!: string;
  descrizione!: string;
  processoCollegato!: string;
  attore!: AttoreDTO;
  ulterioriInfo!: UlterioriInfoDTO;
  eventoRischio!: EventoRischiosoDTO[];
}
